mmpose/models/losses/regression_loss.py

```diff
@@ -117,6 +118,84 @@ def forward(self, output, target, target_weight=None):
         return loss * self.loss_weight
 
 
+@LOSSES.register_module()
+class SoftWingLoss(nn.Module):
+    """Soft Wing Loss 'Structure-Coherent Deep Feature Learning for Robust Face
```

Comment:
Please follow the standard reference format:

https://github.com/open-mmlab/mmpose/blob/228747c298c0801dea093371abeca539d41a7584/mmpose/models/backbones/hrnet.py#L218-L219